store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+            "value": {
+                "name": document_path,
+                "fields": create_document_fields(document_fields),
+                "createTime": timestamp,
+                "updateTime": timestamp
+            },
+            "updateMask": {}
+        }
+    }
+
+
+def create_protobuf_placeholder(project_id: str, database_id: str, collection_path: str, 
```

Ruff (0.11.9)

Unused function argument: `project_id`

(ARG001)